public/computer-lab/js/main.js

```diff
@@ -1 +1,66 @@
-/* global */
+const LAB_WIDTH = 800
+const LAB_HEIGHT = 450
+
+const Lab = {
+  init () {
+    // initialize canvas
+    Lab.canvas = document.getElementById('canvas')
+    Lab.ctx = Lab.canvas.getContext('2d')
+
+    // load  image
+    Lab.image = new Image()
```

Comment:
this line is throwing a StandardJS linter error b/c the linter is unaware of the Image constructor, u can either add the global comment to the top of this file like `/* global Image */` to supress this error, or u could write ur code in a more explicit way. b/c the Image constructor is *technically* part of the global window context u could also write this like 
```js
Lab.image = new window.Image()
```
which would not throw a linter error